draft-ietf-scone-protocol.md

```diff
@@ -206,24 +222,43 @@ to be split between multiple active flows.
 
 The same UDP address tuple might be used for multiple QUIC connections.  A
 single signal might be lost or only reach a single application endpoint.
-Network elements that signal about a flow might choose to send additional
-signals, using connection IDs to indicate when new connections could be
-involved.
+Network elements can apply SCONE advice
+to all flows that include SCONE packets
```

Comment:
```suggestion
to all datagrams that include SCONE packets
```

Is this the intent?

Reply:
"flows" is correct in this context.  The title of the section is "Per-Flow Signal".